### PR TITLE
[Ethan] Non tandem page contents

### PR DESCRIPTION
**Pull request by treybean**
No longer scope tandem contents by page. Instead, use the request parameters. This allows us to place tandem content on any page, not just tandem pages.

Also brought in some capyabara-webkit for a request spec that actually exercises the complete flow of editing content.

### CODE REVIEW

File: app/assets/javascripts/tandem/popup.js [hunk 1]
```diff
@@ -9,34 +10,17 @@
...
-    } else {
-      parent.reload_tandem_content(resource_id,resource_url)
-    }
-  }
+  $('form.tandem_content_editor').on('ajax:success', function(event, content, status, jqXHR){
+    $.get(parent.location.href, function(data, status, jqXHR){
+      var content_id = '#' + content.tag;
 
-  $('.wymeditor').wymeditor({
+      $(content_id, parent.document).html( $(content_id, data).html() );
```
Comment (evizitei) [inline app/assets/javascripts/tandem/popup.js:17]: hmmm, does this work?  Usually I've seen multiple selectors separated by commas in a single string, like this:

http://api.jquery.com/multiple-selector/

I tried this in a console in FF, and this was my result:

```
>>> $(".name, .signatures")
[span.name, h4.name, ul.signatures]
>>> $(".name", ".signatures")
[]
```

File: app/helpers/tandem/pages_helper.rb [hunk 2]
```diff
@@ -73,55 +71,58 @@ def self.included(base)
...
+          })
+        } + content if can? :update, tandem_content
+
+        html_options.merge! id: identifier
+        html_options[:class] ||= ''
+        html_options[:class] << ' tandem_content' if can? :update, tandem_content
+        html_options[:class].strip!
+
+        content_tag(:div, tandem_content, html_options) { content }
+      end
```
Comment (evizitei) [inline app/helpers/tandem/pages_helper.rb:125]: Hmm... this is big enough that we might consider making an object out of it for the next version.

File: app/assets/javascripts/tandem/popup.js (hunk 1 above)
Comment (treybean) [inline app/assets/javascripts/tandem/popup.js:17]: That's not a multiple selector, it's specifying the context to find the selector: http://api.jquery.com/jQuery/

File: app/helpers/tandem/pages_helper.rb (hunk 2 above)
Comment (treybean) [inline app/helpers/tandem/pages_helper.rb:125]: Yeah, I didn't actually change anything here, but agree, this is prime for a refactor. Added to PT.

File: app/assets/javascripts/tandem/popup.js (hunk 1 above)
Comment (evizitei) [inline app/assets/javascripts/tandem/popup.js:17]: Aha, it makes much more sense in that light; hadn't seen the context specifier before

**Pull request merged**